package.json

```diff
@@ -1,20 +1,31 @@
 {
-  "name": "plugin-flex-inqueue-messaging",
+  "name": "plugin-queued-callbacks-and-voicemail",
```

Comment:
Yes, thank you. I will close https://github.com/twilio-labs/plugin-queued-callbacks-and-voicemail/pull/11 since this PR captures that change.